src/core/server/graph/plugins/logger.ts

```diff
@@ -45,7 +45,10 @@ export function logAndReportError(
   const report = ctx.reporter.report(getOriginalError(err) || err, scope);
 
   // Log that we reported an error.
-  ctx.logger.error({ err, report }, "graphql query error");
+  ctx.logger.error(
+    { err, report, traceID: ctx.traceID },
+    "graphql query error"
+  );
```

Comment:
Shouldn't need to add the `traceID` here, as `ctx.logger` already adds a `contextID` which is the same ID.